srml/session/src/lib.rs

```diff
@@ -240,8 +240,7 @@ pub trait SelectInitialValidators<ValidatorId> {
 }
 
 /// Implementation of `SelectInitialValidators` that does nothing.
-pub struct ConfigValidators;
-impl<V> SelectInitialValidators<V> for ConfigValidators {
+impl<V> SelectInitialValidators<V> for () {
```

Comment:
Weird, I removed this some days ago: https://github.com/paritytech/substrate/pull/3037 :man_shrugging: